### DISCUSSION TITLE
Adding Penelope to the project

### DISCUSSION THREAD

**Discussion by Ch4rybd3** [ThePorgs/Exegol#228] [Ideas]
Hey ! 
Huge fan of the project, not really a software dev so I don't want to create a pull request myself to add it, but you could take a look at https://github.com/brightio/penelope, it's a python script that listen for incoming connections and automatically upgrade the shell if possible, the thing is great and would be very cool to see in exegol.

**Comment by ShutdownRepo** [comment]
Thank you for the suggestion!
Wouldn't it be redundant with what [pwncat-cs](https://github.com/calebstewart/pwncat) does?

**Comment by lalvarezt** [comment]
this was added in https://github.com/ThePorgs/Exegol-images/releases/tag/3.1.9
